src/tests/system/pytest.ini

```diff
@@ -2,6 +2,13 @@
 addopts = --strict-markers
 testpaths = tests
 markers =
-    slow: marks tests as slow (deselect with '-m "not slow"')
+    authentication: marks authentication test
+    authorization: marks authorization test
+    cache: marks any cache related test
```

Comment:
Basically all tests with SSSD are related to cache. What purpose do you envision?

Reply:
Tests that explicitly test the functionality of the cache, settings, timeout, and negative cache. Indeed, all tests use it, but we do have some times that explicitly try these settings.

Reply:
It'll primarily reside in this file so yes, this can be removed.

Reply:
This one is hard to grasp for me, since anything that SSSD returns comes from cache. But if you see it beneficial, ok.